src/docs/sphinx/buildGuide/Building GEOSX.rst

```diff
@@ -0,0 +1,70 @@
+Building GEOSX
```

Comment:
A space in the filename `Building GEOSX.rst`?
Someone shall have problems someday.

Reply:
There was another file that did this and I sort of followed suit on autopilot. I will fix both just to be safe.